src/lib.rs

```diff
@@ -7,7 +7,103 @@
 //! * [Documentation](https://tailhook.github.io/gron/gron/index.html)
 
 extern crate rustc_serialize;
+extern crate serde_json;
 
-mod json_struct;
+pub mod for_rustc_serialize;
+pub mod for_serde;
+
+use std::io::{self, Write};
+
+//pub use for_rustc_serialize::json_to_gron as rustc_serialize_json_to_gron;
```

Comment:
Please don't comment out code (at least in final comit). There is a git which keeps history :)